test/test_ops.py

```diff
@@ -1357,5 +1368,87 @@ def test_split_normalization_params(self, norm_layer):
         assert len(params[1]) == 82
 
 
+class TestDropBlock:
+    @pytest.mark.parametrize("seed", range(10))
+    @pytest.mark.parametrize("dim", [2, 3])
+    @pytest.mark.parametrize("p", [0, 0.2, 0.5, 0.8])
+    @pytest.mark.parametrize("block_size", [5, 7, 9, 11])
```

Comment:
Do we need all the combinations of `p` and `block_sizes` given we only assert to a handful of experiments? 

Instead of doing a cartesian product of p x block_sizes, we might want to test just a couple of specific of them to speed the tests.

Reply:
OK, I remove some.